CHANGELOG.md

```diff
@@ -25,6 +25,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.0.2] - 2023-04-07
+### Changed
+- Changing the base image from alpine to opensuse and adding craycli to the image
```

Comment:
You _could_ install the rpm package manager to allow you to install rpms into Alpine.
https://pkgs.alpinelinux.org/package/edge/community/x86_64/rpm
However, you're really utilizing zypper later rather than rpm, so I don't think installing the rpm package manager is the way to go. Thus, I think migrating to OpenSuse is fine.
We started with Alpine because it is a small Linux distribution, and it does not take up much space in a container.